plugins/hubspot/src/blog.ts

```diff
@@ -0,0 +1,169 @@
+import { framer, ManagedCollection, ManagedCollectionField } from "framer-plugin"
+import { fetchAllBlogPosts, HSBlogPost } from "./api"
+import { isDefined, parseJsonToArray } from "./utils"
+import { FieldsById, logSyncResult, MAX_CMS_ITEMS, SyncResult, SyncStatus } from "./cms"
+import { useMutation } from "@tanstack/react-query"
+
+export const PLUGIN_INCLUDED_FIELD_IDS = "hubspotPluginBlogIncludedFieldIds"
```

Comment:
Flipped it. I guess in this case the exclusions will probably be smaller than inclusions. 

Though, also in this case, the (30) blog fields are fixed - which to store is a lot less than 2KB. So it probably won't  make much of a difference here even if a lot more blog fields were added.

Reply:
You can see which fields are not excluded because they have fields with matching ids. If you only want to know if anything changed you could also concatenate all hidden field ids and only store the hash of that value. Then it can never become more than the maximum size

Reply:
Just in case, use this Google Sheets PR for inspiration: https://github.com/framer/plugins/pull/109